### ISSUE TITLE
Role manager - more options for an editor

### ISSUE THREAD

**Issue by JamboMedia** [elementor/elementor#9891]
From this year we are doing our   projects in Elementor.      

At almost all our projects we are facing this problem that clients wants to   change content. So we are giving the client acces to: "edit the content   only"      

This is good, so the client won't break the design. We build are websites based on a PSD file.      

But there are situations that clients want's to do more. 

For example:   
- add new pages. (the client can't add anything from elementor than)   
- also the client want's to duplicate a block.     

In many times, we open the permissions for our customer. This is causing the  big problem that the website is a complete mass because the customer can edit anything.      

We definitly need more options to control this more. 

For example:
 – edit only for specific pages   or   
– lock templates for a user role   or  
 – when an editor creates a new page, he has to add a page template. 

This is   not possible now. (edit only option)      

We like the Role Manager but we need more.      

Are there updates coming soon?

**Comment by webinfinitynz** [comment]
I agree. Many of my clients **won't need to change the design but will need to add menu items**. With the editor role, they can't add menu items, so can't add new pages.

**Comment by black-eye** [comment]
+1
